addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -0,0 +1,81 @@
+jwt.settings.title=JWT Settings
+jwt.toolmenu.settings=JWT Settings...
+ascanrules.jwt.name=JWT Scanner 
+ascanrules.jwt.description=Scanner for finding vulnerabilities in JWT implementations.
+jwt.settings.rsa.header=<html><h3>RSA:<h3></html>
+jwt.settings.general.header=<html><h3>General:<h3></html>
```

Comment:
This doesn't seem to match visually with other options panels.

Reply:
didn't got your point. can you please elaborate.

Reply:
Just using bold and embedding HTML doesn't seem like any of the elements on other ZAP option panels. There is some swing/awt element where a box is drawn that embeds a label to contain or group things, maybe that would be better. I'd have to dig to find the exact details but I just remember when I was going through the options panels in the GUI when I saw this I was like: "This is different"

Reply:
![image](https://user-images.githubusercontent.com/7570458/81444412-1471e180-9145-11ea-8db1-d3a0bbaadff9.png)

Reply:
hmm.. makes sense. let me figure out what component it is. will do the changes.